src/main/resources/templates/snowflake/shared/create-table-evolve-schema.ssp

```diff
@@ -10,7 +10,7 @@
 #if (tableDiff.existsInDestination && tableDiff.existsInSource)
     #if (tableDiff.allChangesAreCompatible())
         ALTER TABLE ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${tableDiff.destinationName} ADD COLUMN ${unescape(tableDiff.columnDDL(typeMapping))};
-    #elseif (!tableDiff.allChangesAreCompatible())
+    #else
```

Comment:
when we use #else we are getting error : org.fusesource.scalate.InvalidSyntaxException: The #elseif cannot come after the #else inside the #if at 18.1 in junit.

But as per docs https://scalate.github.io/scalate/documentation/ssp-reference.html#_code__if__code_  its correct syntax.  I tried upgrading version of library "scalate-core" to latest but still getting error. May be the library is different. 
I am reverting this change. 
Please let me know ur feedback.